
# I am unable to open jupitor notebook

Traceback (most recent call last):
File "C:\Users\azees\anaconda3\Scripts\jupyter-notebook-script.py", line 6, in
from notebook.notebookapp import main
File "C:\Users\azees\anaconda3\lib\site-packages\notebook\notebookapp.py", line 43, in
from jinja2 import Environment, FileSystemLoader
File "C:\Users\azees\anaconda3\lib\site-packages\jinja2_init_.py", line 12, in
from .environment import Environment
File "C:\Users\azees\anaconda3\lib\site-packages\jinja2\environment.py", line 25, in
from .defaults import BLOCK_END_STRING
File "C:\Users\azees\anaconda3\lib\site-packages\jinja2\defaults.py", line 3, in
from .filters import FILTERS as DEFAULT_FILTERS # noqa: F401
File "C:\Users\azees\anaconda3\lib\site-packages\jinja2\filters.py", line 13, in
from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (C:\Users\azees\anaconda3\lib\site-packages\markupsafe_init_.py)
actually for my project i am trying to import tensorflow library ..i imported tf using comand prompt then onwards i am getting errors... even jupitornotebook is not opening

        